.github/workflows/run-tests.yml

```diff
@@ -55,10 +55,20 @@ jobs:
         with:
           php-version: 7.4
           coverage: none
-        
+          tools: composer:v2
+
       - name: Install dependencies
         run: |
           composer remove phpro/grumphp vimeo/psalm --no-update --dev
           composer update --prefer-dist --no-suggest --no-progress
 
-      - run: composer check-style
+      - run: composer fix-style
+        continue-on-error: true
+
+      # Revert modifications so they don't get commited 💥
+      - run: git checkout -- composer.json
```

Comment:
There's `composer remove` step to avoid installing too many unnecessary dependencies, this must be rolled back before a potential commit hapens